.github/workflows/fast_testing.yml

```diff
@@ -0,0 +1,34 @@
+name: fast_testing
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - 'master'
+    tags:
+      - '*'
```

Comment:
Can we do the same as in https://github.com/tarantool/tuple-keydef/pull/11 ?

1. Run on push to any branch (I want to create developer branches in the repository and see CI results before opening a pull request).
2. Skip duplicated push and pull request jobs.

Don't know, whether we should run testing on tags or not.

Reply:
Fixed.